packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -162,8 +162,7 @@ function RevisionsButtons( {
 						aria-current={ isSelected }
 					>
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> revisions icon --> check for individual revisions button.

**Note:** This has some overridden styles, so no difference in before and after.

| Before | After     |
| :-------- | :------- |
| <img width="625" alt="revisions-button-before" src="https://github.com/user-attachments/assets/368bbd6b-96a3-427b-a93c-9d6ea8f4fdfc"> | <img width="625" alt="revisions-button-after" src="https://github.com/user-attachments/assets/169d6af0-9e67-4d82-8578-2efc5e447f0b"> |